keyboards/rotary_numpad/readme.md

```diff
@@ -0,0 +1,60 @@
+# Rotary Numpad
+
+
+![rotary_numpad](https://i.imgur.com/GwEH4WOh.jpeg)
+
+This is a personal project of mine where I wanted a number pad that I can put on the left side of keyboard that doesn't have a numberpad. I fell in love with the feel of rotary encoders on my 3d printer, so added three to be able to play around with.
+
+* Keyboard Maintainer: [rabijl](https://github.com/rabijl)
+* Hardware Supported: Sparkfun Pro Micro 5V (atmega32u4)
+* Hardware Availability: https://www.sparkfun.com/products/12640
+
+## How to build from scratch
+
+I handsoldered gateron blue switches to a 3D printed case. The design is modified from the [Ultipad](https://www.thingiverse.com/thing:4248565). \
+I used a sparkfun Pro Micro as my microcontroller and the pinout can be found in the `info.json`. I found this [site](https://golem.hu/article/pro-micro-pinout/) \
+to be very useful to identify different pins. You can find useful information on rotary encoders from the qmk docs [here](https://docs.qmk.fm/#/reference_info_json?id=encoders) and [here](https://docs.qmk.fm/#/feature_encoders).
```

Comment:
All of this suggests that it is a handwired board, which if it is, should be located under the `keyboards/handwired` directory.